test/integration/utils/taskruns.go

```diff
@@ -68,14 +72,19 @@ func (t *TestBuild) GetTRTillDesiredReason(buildRunName string, reason string) e
 		pollTRTillCompletion = func() (bool, error) {
 
 			trReason, err := t.GetTRReason(buildRunName)
+			// Not found means that we may not have a TaskRun for this BuildRun yet
+			if kerrors.IsNotFound(err) {
+				fmt.Printf("could not find TaskRun for BuildRun %s", buildRunName)
```

Comment:
Sending the logs to stdout pollutes the whole testing, for getting insights on this use case(if taskrun exists or not), we better call first `GetTaskRunFromBuildRun` in the test case, and then we assert for nil in the error, e.g.:

```
tr, err := tb.GetTaskRunFromBuildRun(buildRunObject.Name)
Expect(err).To(BeNil())
```
pls remove the fmt.Printf line